### DISCUSSION TITLE
articles/wangxiaobo

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#122] [Comments]
https://hutusi.com/articles/wangxiaobo

 4月11日，中午的时候突然觉得这是个特殊的日子，想了半天没想出来有什么意义。晚上突然想到可能是王小波的忌日，上网一查果如其然。十年前的今天，天也是这么黑，风也是这么大，干燥而阴冷的空气中弥漫着一中不详的预兆，一个伟大而智慧的头脑停止思考了。（不过王小波死是在11日凌晨）古希腊哲学家泰利斯有一次因为仰头观察天象，却...

**Comment by boosyang** [comment]
# 点赞  👍

**Reply by hutusi** [reply]
二十年前写的了，时间过得好快啊。
